config/routes.rb

```diff
@@ -18,4 +18,6 @@
 
 	resources :subjects
 	resources :teaching_classes
+	resources :subject_and_classes, only: [:create]
+	delete '/subject_and_classes', to: 'subject_and_classes#remove_subject_and_classes'
```

Comment:
@josephelias-rubyians  Can you try another way without using setting the url? This is non-resource way

you can try within the resources ?

[https://guides.rubyonrails.org/routing.html#adding-collection-routes](https://guides.rubyonrails.org/routing.html#adding-collection-routes)

Reply:
sure will check this.